runtime-light/stdlib/exit/exit.cpp

```diff
@@ -0,0 +1,66 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/stdlib/exit/exit.h"
+
+#include <cstdint>
+
+#include "runtime-core/utils/kphp-assert-core.h"
+#include "runtime-light/component/component.h"
+#include "runtime-light/header.h"
+#include "runtime-light/streams/streams.h"
+#include "runtime-light/utils/context.h"
+
+namespace {
+
+int32_t ob_merge_buffers() {
+  Response &response{get_component_context()->response};
+  php_assert(response.current_buffer >= 0);
+
+  int32_t ob_first_not_empty{};
+  while (ob_first_not_empty < response.current_buffer && response.output_buffers[ob_first_not_empty].size() == 0) {
+    ++ob_first_not_empty;
+  }
+  for (auto i = ob_first_not_empty + 1; i <= response.current_buffer; i++) {
+    response.output_buffers[ob_first_not_empty].append(response.output_buffers[i].c_str(), response.output_buffers[i].size());
+  }
+  return ob_first_not_empty;
+}
+
+} // namespace
+
+task_t<void> shutdown_script() noexcept {
+  auto &component_ctx{*get_component_context()};
+  const auto standard_stream{component_ctx.standard_stream()};
+  if (standard_stream == INVALID_PLATFORM_DESCRIPTOR) {
+    component_ctx.poll_status = PollStatus::PollFinishedError;
+    co_return;
+  }
+
+  const auto ob_total_buffer{ob_merge_buffers()};
```

Comment:
Is it good idea to set `auto` everywhere? As for me it makes the code harder to read